Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@model CatalogueModel
+@{
+    ViewData["Title"] = "Catalogue";
+}
+
+<h1>@ViewData["Title"]</h1>
+
+@if (Model.Catalogue == null)
```

Comment:
I would expect the catalogue to be empty, not null, if there are no books. If it can be `null` then please mark it as nullable (although in this case I would change the logic if possible to make sure it's not null) 🙂